python/ray/tune/logger.py

```diff
@@ -372,12 +372,21 @@ def update_config(self, config):
     def close(self):
         for _logger in self._loggers:
             _logger.close()
-        self._log_syncer.sync_down()
```

Comment:
hm why remove?

Reply:
cuz we call close() for failed trials and we dont want to try syncing down from a failed node. so i just moved the call out to the relevant places at the call-sites